src/aalwines/model/Router.cpp

```diff
@@ -325,9 +326,19 @@ namespace aalwines
             {
                 s << ",\n";
             }
-            s << "\t\t\t\t\"" << in << "\"";
+            s << "\t\t\t\t\"" << in.first << "\": [";
+            bool first_label = true;
+            for (const auto& label : in.second) {
+                if (first_label) {
+                    first_label = false;
+                } else {
+                    s << ",";
+                }
+                s << label;
```

Comment:
strings are missing quotes
```suggestion
                s << "\"" << label << "\"";
```